src/db/migrations/3_CreateConfigTable.js

```diff
@@ -0,0 +1,31 @@
+const debug = require("../debug");
+
+class CreateConfigTable extends require("../Migration") {
+  async body(index, database) {
+    let done = false;
+    let sql = await this.sql();
+
+    // await sql.schema.dropTableIfExists("test");
+
+    if (!(await sql.schema.hasTable("scraper_config"))) {
+      await sql.schema.createTable("scraper_config", async (table) => {
+        table.increments("id").primary();
+        table.string("name").notNullable().unique();
+        table.string("address").notNullable().unique();
```

Comment:
Make it not unique. We may want too fix a table setting a new version for the name without changing the address